### DISCUSSION TITLE
Change behaviour of Shift+Enter

### DISCUSSION THREAD

**Discussion by desk7** [vslinko/obsidian-outliner#52] [Archive]
Hello,
I think that the behaviour of Shift+Enter shortcut should be different from standard Obsidian and should be like in Logseq:

- in Obsidian it starts a new line without lists
- in Logseq, as it should be, it starts a new line belonging to the _same_ item

I don't really know if you should implement it or if it should be a global Obsidian change. It's surely an important thing to do.

**Comment by vslinko** [comment]
@desk7 Thank you for your idea! It's already in the backlog. Please follow #38

**Comment by vslinko** [comment]
It's released! https://github.com/vslinko/obsidian-outliner/releases/tag/1.2.0
